core/server/src/log/logger.rs

```diff
@@ -253,8 +255,48 @@ impl Logging {
             let base_directory = PathBuf::from(base_directory);
             let logs_subdirectory = PathBuf::from(config.path.clone());
             let logs_path = base_directory.join(logs_subdirectory.clone());
-            let file_appender =
-                tracing_appender::rolling::hourly(logs_path.clone(), IGGY_LOG_FILE_PREFIX);
+
+            if let Err(e) = std::fs::create_dir_all(&logs_path) {
+                tracing::warn!("Failed to create logs directory {:?}: {}", logs_path, e);
```

Comment:
Let's import `warn` on top, next to `error, info`. Also, let's use string interpolation for all the messages, e.g. here and a few lines below.